config/examples/Elegoo/Neptune-2/Configuration_adv.h

```diff
@@ -2337,7 +2339,7 @@
 //#define NO_TIMEOUTS 1000 // Milliseconds
 
 // Some clients will have this feature soon. This could make the NO_TIMEOUTS unnecessary.
-#define ADVANCED_OK
+//#define ADVANCED_OK
```

Comment:
These are all for proper serial communucation, leaving it on default is unwarranted IMO, it'll just increase amount of questions "why octoprint lose connection?"

Reply:
reverted in the latest. Thanks for helping me understand better what this does.